api/manifest.proto

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Tetrate
```

Comment:
I didn't import the source history of this file as there weren't many revisions and the last was nearly 2 years ago https://github.com/tetratelabs/getenvoy-package/commits/master/api/manifest.proto